Bottom: a7817d70af514c5b2a8ab32d980415b74d41c71e
Top:    9622952f0628e8305150e4c1c747c231124f7f63
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-03 19:35:49 +0100

Refresh of attempt-at-fixing-theming

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 1cc1567..075732c 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1208,8 +1208,10 @@ get_widget_styles (GtkStyle **menu_bar_style,
   gtk_container_add (GTK_CONTAINER (tool_button), label);
 
   /* Extract the theme's style for a toolbar button */
-  gtk_widget_ensure_style (label);
+  gtk_widget_ensure_style (button);
   *tool_button_style = gtk_widget_get_style (tool_button);
+
+  gtk_widget_ensure_style (label);
   *tool_button_label_style = gtk_widget_get_style (label);
 
   gtk_widget_destroy (window);
